config/runtime/boot/fastboot.jinja2

```diff
@@ -0,0 +1,29 @@
+- deploy:
+    images:
+      image:
+        url: '{{ node.artifacts.kernel }}'
+      dtb:
+        url: '{{ node.artifacts.dtb }}'
+      ramdisk:
+        url: 'http://storage.kernelci.org/images/rootfs/buildroot/buildroot-baseline/20230703.0/{{ brarch }}/rootfs.cpio.gz'
+     postprocess:
+      docker:
+        image: ghcr.io/mwasilew/docker-mkbootimage:master
+        steps:
+        - mkbootimg --header_version 2 --kernel Image --dtb qcs6490-rb3gen2.dtb --cmdline "earlycon clk_ignore_unused pd_ignore_unused audit=0" --ramdisk rootfs.cpio.gz --output boot.img
+    to: downloads
+
+- deploy:
+    images:
+      boot:
+        url: 'downloads://boot.img'
+    timeout:
+      minutes: 20
+    to: download
+
+- boot:
+    prompts:
+    - root@(.*):[/~]#
+    timeout:
+      minutes: 15
```

Comment:
```suggestion
    failure_retry: 3
    timeout:
      minutes: 10
    timeouts:
      bootloader-commands:
        minutes: 3
      auto-login-action:
        minutes: 2
```

This would make it aligned with https://github.com/kernelci/kernelci-core/pull/2558, allowing multiple boot attempts in case something goes wrong (either on the board or on the LAVA side).

Reply:
fixed.